### DISCUSSION TITLE
Having trouble syncing with Dropbox

### DISCUSSION THREAD

**Discussion by peshala-prabhapoorna** [super-productivity/super-productivity#3660] [Q&A] [answered]
I use Super Productivity on Linux and Android. So I synced data between the devices using Dropbox. Last night I reinstalled my OS and now syncing isn't working. After I paste in an save the auth code from Dropbox, a notification says "Access token generated from auth code". Then when I try to sync, an error appears that says "Unknown Sync Error: Error: Invalid rev given".

I feel like I've screwed up somewhere. Can  someone help me out?

update: tried using the web app, but got the same error after saving the auth code.

desktop app:
![Screenshot From 2024-11-06 10-06-40](https://github.com/user-attachments/assets/8087d03c-a986-42fc-ae3f-5c029b363bb7)
![Screenshot From 2024-11-06 10-05-53](https://github.com/user-attachments/assets/e0545e56-56f7-4568-a8f9-ac0b20ad8090)

webapp:
![Screenshot From 2024-11-06 10-18-20](https://github.com/user-attachments/assets/df724e29-52dc-4a57-be77-192d62cd290a)

**Comment by peshala-prabhapoorna** [comment, marked as answer]
This is an issue and is reported in #3661. It is now solved in https://github.com/johannesjo/super-productivity/releases/tag/v10.1.1.

**Comment by johannesjo** [comment]
Thanks for reporting this behavior @peshala-prabhapoorna !
